context/wallet.tsx

```diff
@@ -0,0 +1,46 @@
+import { WalletAdapterNetwork, WalletError } from '@solana/wallet-adapter-base';
+import {
+  ConnectionProvider,
+  WalletProvider,
+} from '@solana/wallet-adapter-react';
+import { WalletModalProvider } from '@solana/wallet-adapter-react-ui';
+import {
+  LedgerWalletAdapter,
+  PhantomWalletAdapter,
+  SolflareWalletAdapter,
+  SolletWalletAdapter,
+} from '@solana/wallet-adapter-wallets';
+import { clusterApiUrl } from '@solana/web3.js';
+import { ChildrenProps as Props } from '@utils/interface/global-interface';
+import React, { FC, ReactNode, useCallback, useMemo } from 'react';
+
+const WalletContextProvider: FC<{ children: ReactNode }> = ({
+  children,
+}: Props) => {
+  // Todo: change the network based on the node env
+  const network = WalletAdapterNetwork.Devnet;
```

Comment:
Based from the todo that you have above, we can probably add a wallet config under the `config` folder where you can include this. Not needed at the moment, but just a quick note. Can be defaulted to devnet if nothing is written on env file. 

ps. not sure how to add a normal comment on the code so I'm just adding this as a review comment. lol. Just resolve this if you read it :D